_meta/beat.yml

```diff
@@ -0,0 +1,7 @@
+################### Mysqlbeat Configuration Example #########################
+
+############################# Mysqlbeat ######################################
+
+mysqlbeat:
+  # Defines how often an event is sent to the output
+  period: 1s
```

Comment:
1s is probably a little excessive. How about 10s?

Reply:
Yep in next PR, changed to 1m interval